src/router/index.ts

```diff
@@ -42,4 +53,10 @@ const router = createRouter({
   ],
 });
 
+// router.beforeEach((to, from) => {
+//   const userStore = useUserStore();
+
+//   if (!userStore.isLoggedIn && to.name === "LoginPage") return "/login";
+// });
+
```

Comment:
If it's unused code it's better to remove it. Once you'll need it again it will be preserved in git history.